manifests/3.1.0/opensearch-3.1.0.yml

```diff
@@ -47,6 +47,8 @@ components:
     platforms:
       - linux
       - windows
+    depends_on:
+      - common-utils
```

Comment:
Adding k-NN [maintainers](https://github.com/opensearch-project/k-NN/blob/main/MAINTAINERS.md) to confirm this dependency and also recommend if there are more.
@naveentatikonda @navneet1v @jmazanec15

Reply:
Given their version bump PR is failing on that, I'm pretty confident. :) 
 - https://github.com/opensearch-project/k-NN/pull/2695

Reply:
yes - see https://github.com/opensearch-project/k-NN/blob/main/build.gradle#L357.

Security also, but maybe everyone depends on that

Reply:
Security should be a test dependency, not a build dependency right?

Reply:
right!

Reply:
> yes - see https://github.com/opensearch-project/k-NN/blob/main/build.gradle#L357.
> 
> Security also, but maybe everyone depends on that

That dependency on the next line looks fishy, I don't trust their maintainer. ;)
https://github.com/opensearch-project/k-NN/blob/c876632466cd6fb37090663b13e63b6435bd3d5e/build.gradle#L358